relay-event-normalization/src/normalize/span/tag_extraction.rs

```diff
@@ -312,6 +316,12 @@ fn extract_shared_tags(event: &Event) -> BTreeMap<SpanTagKey, String> {
         }
         if let Some(country_code) = user.geo.value().and_then(|geo| geo.country_code.value()) {
             tags.insert(SpanTagKey::UserCountryCode, country_code.to_owned());
+            if let Some(subregion) = Subregion::from_iso2(&country_code.as_str()) {
+                let numerical_subregion = subregion as u8;
```

Comment:
If you cast, the enum should be the correct [repr](https://doc.rust-lang.org/reference/items/enumerations.html#explicit-discriminants).